lib/fathead/python_six/output.txt

```diff
@@ -0,0 +1,142 @@
+and subclasses provide a __next__ method. In Python 2, Iterator
+has one method: next. It simply delegates to __next__. An alternate
+way to do this would be to simply alias next to __next__. However,
+this interacts badly with subclasses that override
+__next__. Iterator is empty on Python 3. (In fact, it is just
+aliased to object.)</p></section>	https://pythonhosted.org/six//index.html
+wraps	A										<section class="prog__container"><p>This is exactly the functools.wraps() decorator, but it sets the
+__wrapped__ attribute on what it decorates as functools.wraps()
+does on Python versions after 3.2.</p></section>	https://pythonhosted.org/six//index.html
+exec_	A										<section class="prog__container"><p>Execute code in the scope of globals and locals.  code can be a
+string or a code object.  If globals or locals are not given, they will
+default to the scope of the caller.  If just globals is given, it will also
+be used as locals.</p><pre><code>\nNote\nPython 3’s exec() doesn’t take keyword arguments, so calling\nexec() with them should be avoided.\n</code></pre></section>	https://pythonhosted.org/six//index.html
+print_	A										<section class="prog__container"><p>Print args into file.  Each argument will be separated with sep and
+end will be written to the file after the last argument is printed.  If
+flush is true, file.flush() will be called after all data is written.</p><pre><code>\nNote\nIn Python 2, this function imitates Python 3’s print() by not\nhaving softspace support.  If you don’t know what that is, you’re probably\nok. :)\n</code></pre></section>	https://pythonhosted.org/six//index.html
+raise_from	A										<section class="prog__container"><p>Raise an exception from a context.  On Python 3, this is equivalent to
+raise exc_value from exc_value_from.  On Python 2, which does not support
+exception chaining, it is equivalent to raise exc_value.</p></section>	https://pythonhosted.org/six//index.html
+reraise	A										<section class="prog__container"><p>Reraise an exception, possibly with a different traceback.  In the simple
+case, reraise(*sys.exc_info()) with an active exception (in an except
+block) reraises the current exception with the last traceback.  A different
+traceback can be specified with the exc_traceback parameter.  Note that
+since the exception reraising is done within the reraise() function,
+Python will attach the call frame of reraise() to whatever traceback is
+raised.</p></section>	https://pythonhosted.org/six//index.html
+with_metaclass	A										<section class="prog__container"><p>Create a new class with base classes bases and metaclass metaclass.  This
+is designed to be used in class declarations like this:</p><pre><code>from six import with_metaclass\n\nclass Meta(type):\n    pass\n\nclass Base(object):\n    pass\n\nclass MyClass(with_metaclass(Meta, Base)):\n    pass\n\n</code></pre></section>	https://pythonhosted.org/six//index.html
```

Comment:
@iamrajhans looks like there is a line-break here in the middle of the abstract? Please investigate where that's coming from. We can't have any unescaped newlines in the `output.txt`